### PR TITLE
NSA-3637- adding service_type for saml signout

### PR DESCRIPTION
**Pull request by jovin-joy**

### CODE REVIEW

File: src/infrastructure/utils/index.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ const { getOrganisationsAssociatedWithUser, getPendingRequestsForApproval } = re
 const APPROVER = 10000;
 
 const isLoggedIn = (req, res, next) => {
-  if (req.isAuthenticated() || req.baseUrl === '/signout') {
+  const serviceType = req.query? req.query.service_type:null;
+  if (req.isAuthenticated() || (req.baseUrl === '/signout' && serviceType !== 'saml') {
```
Comment (thedapperdev) [inline src/infrastructure/utils/index.js:10]: do we need an extra `)` here?

Comment (jovin-joy) [reply src/infrastructure/utils/index.js]: good spot Jordan, done

Comment (jovin-joy) [reply src/infrastructure/utils/index.js]: thanks for that

**Review by thedapperdev — APPROVED** [review]

File: src/infrastructure/utils/index.js (hunk 1 above)
Comment (thedapperdev) [reply src/infrastructure/utils/index.js]: no worries 😺

**Pull request merged**